app/frontend/packs/application.scss

```diff
@@ -1,4 +1,26 @@
 @charset "UTF-8";
+
```

Comment:
It would be nice to split this out but not a blocker for merge. created https://github.com/pulibrary/notebook/issues/9